### PR TITLE
Add support of the NonSemanticsDebugBreak

### PR DESCRIPTION
**Pull request by jiaolu**
DebugBreak would generate system s_sethalt 1 instruction

### CODE REVIEW

**Comment by qiaojbao** [comment]
retest this please

**Comment by jiaolu** [comment]
retest

File: llpc/test/shaderdb/general/PipelineCs_DebugBreak_intrinsic.pipe [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+[CsSpirv]
+; SPIR-V
+; Version: 1.6
+; Generator: Khronos Glslang Reference Front End; 11
+; Bound: 20
+; Schema: 0
+               OpCapability Shader
+			   OpExtension "SPV_KHR_non_semantic_info"
+          %2 = OpExtInstImport "GLSL.std.450"
+		  %20 = OpExtInstImport "NonSemantic.DebugBreak"
```
Comment (Flakebi) [inline llpc/test/shaderdb/general/PipelineCs_DebugBreak_intrinsic.pipe:13]: There are some tabs here that should probably be spaces.

**Review by Flakebi — COMMENTED** [review]
Nice, LGTM

**Comment by amdvlk-admin** [comment]
### Test summary for commit c4d5b5f4394f9f10e966d7317b98fa92aed8d9ed
<details><summary>CTS tests (Failed: 0/211534)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 39251/70518 (55.7%)</li>
<li>Failed: 0/70518 (0.0%)</li>
<li>Not Supported: 31267/70518 (44.3%)</li><li>Warnings: 0/70518 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 39499/70507 (56.0%)</li>
<li>Failed: 0/70507 (0.0%)</li>
<li>Not Supported: 31008/70507 (44.0%)</li><li>Warnings: 0/70507 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 9.0, Gfx10</summary><ul>

<li>Passed: 36346/70509 (51.5%)</li>
<li>Failed: 0/70509 (0.0%)</li>
<li>Not Supported: 34163/70509 (48.5%)</li><li>Warnings: 0/70509 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by qiaojbao** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 0be786463745ff5014b53cc89ee3745b5a0af872
<details><summary>CTS tests (Failed: 0/205094)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 36341/68149 (53.3%)</li>
<li>Failed: 0/68149 (0.0%)</li>
<li>Not Supported: 31808/68149 (46.7%)</li><li>Warnings: 0/68149 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 36569/68149 (53.7%)</li>
<li>Failed: 0/68149 (0.0%)</li>
<li>Not Supported: 31580/68149 (46.3%)</li><li>Warnings: 0/68149 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 9.0, Gfx10</summary><ul>

<li>Passed: 34225/68796 (49.7%)</li>
<li>Failed: 0/68796 (0.0%)</li>
<li>Not Supported: 34571/68796 (50.3%)</li><li>Warnings: 0/68796 (0.0%)</li>
</ul></details>


</ul></details>

**Review by nhaehnle — APPROVED** [review]

**Pull request merged**